metaci/testresults/tests/test_robot_importer.py

```diff
@@ -514,3 +514,26 @@ def test_gus_bus_payload_metadata():
             "key1": True,
             "key2": "hello, world",
         }
+
+
+@responses.activate
+@pytest.mark.django_db
+def test_gus_bus_bad_payload_metadata():
+    """Verify that all build-specific metadata is added to the payload"""
+    flowtask = FlowTaskFactory()
+    flowtask.build_flow.build.org = OrgFactory()
+    flowtask.build_flow.build.repo.metadata = {"key1": True, "key2": "hello, world"}
+    responses.add(
+        "POST",
+        f"{settings.METACI_RELEASE_WEBHOOK_URL}/test-results/",
+        json={"success": True},
+        status=422,
```

Comment:
Don't use fake data in tests that isn't realistic. If we got a 422 from gus-bus then the body would have information about what was wrong in the input, not `{"success": True}`